test-fixtures/src/main/kotlin/io/forestframework/testfixtures/AbstractForestIntegrationTest.kt

```diff
@@ -88,10 +88,10 @@ suspend fun HttpClient.get(port: Int, uri: String, headers: Map<String, String>
     }
 }
 
-suspend fun HttpClient.send(httpMethod: HttpMethod, port: Int, uri: String, headers: Map<String, String> = emptyMap()) = awaitResult<HttpClientResponse> { handler ->
+suspend fun HttpClient.delete(port: Int, uri: String, headers: Map<String, String> = emptyMap()) = awaitResult<HttpClientResponse> { handler ->
```

Comment:
```
suspend fun HttpClient.delete(port: Int, uri: String, headers: Map<String, String> = emptyMap()) = send(DELETE, port, uri, headers)

suspend fun HttpClient.get(port: Int, uri: String, headers: Map<String, String> = emptyMap()) = send(GET, port, uri, headers)
```

Reply:
oh... i didn't realize it until right now